### PR TITLE
Sqa bug fixes

### LINKED ISSUE #131: Add TNM findings endpoint to backend
## Definition of done
TNM findings in the disease edit UI should be pulled from the backend endpoint. The fakebackend already has the endpoint, but it need to be added to the backend itself, with the endpoint pulling the data from a local json dataset.

### PR DESCRIPTION
**Pull request by belkassaby**
This PR will fix #131, #127, #129, #116 and #132

### CODE REVIEW

**Comment by belkassaby** [comment]
this branch has been deployed to http://35.227.26.28/

**Review by ielis — APPROVED** [review]
Looks good to me, thanks!

**Review by iimpulse — CHANGES_REQUESTED** [review]

File: frontend/src/app/component/phenopacket/phenotypic-feature/phenotypic-feature.component.ts [hunk 1]
```diff
@@ -173,11 +163,21 @@ export class PhenotypicFeatureComponent implements AfterViewInit, OnInit, OnChan
...
+            phenotypicFeature.modifiers.forEach(modifier => {
+                result.push(modifier.id);
+            });
+            return result.join(',');
+        }
+        return '';
+    }
+    getEvidences(phenotypicFeature: PhenotypicFeature) {
+        if (phenotypicFeature.evidences) {
+            const result = [];
```
[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/src/app/component/phenopacket/phenotypic-feature/phenotypic-feature.component.ts:176]: the result object which is an array is never re-assigned/re-initialized: it's initialized only once, right above. Then I call .push on the array object which doesn't re-assign the object. It's still the same object reference. If I was to later in the code do something like `code = ['new value', 'new valu2]; ` then yeah I should use let. I have a typescript linting which runs before each commit and throws errors if I was to re-initialize such a const variable. See [const/var/let](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.) for more info.

[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/src/app/component/phenopacket/phenotypic-feature/phenotypic-feature.component.ts:176]: Also i didn't want to declare it at the class level, as it's used only in that bit of code to display a string of evidences separated by commas.

File: frontend/src/app/component/shared/edit/disease-edit/disease-edit.component.ts [hunk 2]
```diff
@@ -105,23 +193,43 @@ export class DiseaseEditComponent implements OnInit, OnDestroy {
...
+            this.disease.diseaseStage = [];
+            this.disease.diseaseStage.push(new OntologyClass(event.node.key, event.node.label));
             this.diseaseChange.emit(this.disease);
         }
     }
-    updateFindingStages(findings) {
+    updateLaterality(laterality) {
         if (this.disease) {
-            this.disease.clinicalTnmFinding = findings;
+            console.log(laterality);
```
[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/src/app/component/shared/edit/disease-edit/disease-edit.component.ts:206]: I'll remove these directly on the dev branch if it's ok.

[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/src/app/component/shared/edit/disease-edit/disease-edit.component.ts:206]: actually I'll commit these in the current branch im working on.

**Pull request merged**